app/views/schools/advice/long_term/_long_term_trend.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= i18n_prefix = "advice_pages.#{fuel_type}_long_term"
+    render 'schools/advice/section_title',
+           section_id: 'long-term-trends',
+           section_title: t("#{i18n_prefix}.analysis.long_term_trends.title") %>
+
+<p><%= t("#{i18n_prefix}.analysis.long_term_trends.summary") %></p>
+
+<% if analysis_dates.months_of_data > 23 %>
+  <%# i18n-tasks-use t('advice_pages.electricity_long_term.charts.electricity_longterm_trend.subtitle_html') %>
+  <%# i18n-tasks-use t('advice_pages.gas_long_term.charts.gas_longterm_trend.subtitle_html') %>
+  <% suffix = :_academic_year if Flipper.enabled?(:school_academic_year_comparisons) %>
```

Comment:
Should pass in current user here. That allows us to switch on the feature for admins (for example) but not other users.